uberfire-extensions/uberfire-wires/uberfire-wires-core/uberfire-wires-core-grids/src/main/java/org/uberfire/ext/wires/core/grids/client/widget/grid/impl/BaseGridWidget.java

```diff
@@ -313,9 +334,11 @@ protected void drawWithoutTransforms(Context2D context,
         }
 
         //Then render to the canvas
+        currentTimeMillis = log(" - Pre- super.drawWithoutTransforms()");
         super.drawWithoutTransforms(context,
                                     alpha,
                                     bb);
+        log(" - Post- super.drawWithoutTransforms() - " + (System.currentTimeMillis() - currentTimeMillis) + "ms");
```

Comment:
log(" - Post- super.drawWithoutTransforms() - ", currentTimeMillis);

 private static long log(final String message, long currentTimeMillis) {
        if (LOGGER.isLoggable(Level.FINEST)) {
            LOGGER.log(Level.FINEST, message + (System.currentTimeMillis() - currentTimeMillis) + "ms");
        }
        return System.currentTimeMillis();
    }

Let's reduce boilerplate code, if possible

Reply:
Changed in new commit.